src/chttpd_view.erl

```diff
@@ -33,8 +33,7 @@ multi_query_view(Req, Db, DDoc, ViewName, Queries) ->
         Acc1
     end, VAcc1, ArgQueries),
     {ok, Resp1} = chttpd:send_delayed_chunk(VAcc2#vacc.resp, "\r\n]}"),
-    {ok, Resp2} = chttpd:end_delayed_json_response(Resp1),
-    {ok, Resp2#vacc.resp}.
+    chttpd:end_delayed_json_response(Resp1).
```

Comment:
returning Resp2#vacc.resp matches what design_doc_view returns and both are called only from handle_view_req.

I've reproduced this crash and confirmed this change fixes it, but it feels like something deeper is wrong and the true fix is elsewhere.